packages/analytics/src/session.ts

```diff
@@ -74,7 +74,7 @@ export const createSession = (
     loggedInUserId: string = ''
 ): Session => ({
     user: {
-        user_id: cookie.parse(document.cookie).giphy_pbid,
+        user_id: cookie.parse(document ? document.cookie : ({} as any)).giphy_pbid,
```

Comment:
We'd never fire a pingback on the server, but maybe for testing we might not have a `document`